hack/rpm-deps.sh

```diff
@@ -63,6 +63,19 @@ testimage_main="
   util-linux
   which
 "
+testimage_main_s390x="
+  device-mapper
+  e2fsprogs
+  iputils
+  nmap-ncat
+  procps-ng
+  qemu-img-${QEMU_VERSION}
+  sevctl
+  tar
+  targetcli
+  util-linux
+  which
+"
```

Comment:
This is now the same as `testimage_main` so there's no point in defining it. Make sure you update the caller below as well.

Reply:
Good catch. I have removed that bit and rebased the PR.
I also needed to move the `qemu-kvm-device-usb-redirect` back to arch specific `launcherbase` definitions, but running `make rpm-deps` after that did not change anything except the sandbox hash.